test/other-issues/lazy-count/lazy-count.ts

```diff
@@ -266,6 +266,33 @@ describe("other issues > lazy count", () => {
             }),
         ))
 
+    it("run count query when offset is greater than total entities", () =>
```

Comment:
Could we have another test which verifies this works with `skip` and joined relations as well?